src/Ocelot/DependencyInjection/OcelotBuilder.cs

```diff
@@ -114,9 +112,11 @@ public OcelotBuilder(IServiceCollection services, IConfiguration configurationRo
             Services.TryAddSingleton<IHttpHandlerOptionsCreator, HttpHandlerOptionsCreator>();
             Services.TryAddSingleton<IDownstreamAddressesCreator, DownstreamAddressesCreator>();
             Services.TryAddSingleton<IDelegatingHandlerHandlerFactory, DelegatingHandlerHandlerFactory>();
-            Services.TryAddSingleton<ICacheKeyGenerator, DefaultCacheKeyGenerator>();
+            
```

Comment:
Line 82 is also redundant for the `IRegionCreator` interface.

Reply:
ok